src/material/radio/radio.ts

```diff
@@ -328,38 +328,13 @@ const _MatRadioButtonMixinBase:
         mixinDisableRipple(mixinTabIndex(MatRadioButtonBase));
 
 /**
- * A Material design radio-button. Typically placed inside of `<mat-radio-group>` elements.
+ * Base class with all of the `MatRadioButton` functionality.
+ * @docs-private
```

Comment:
FWIW: `docs-private` should not be needed here as the class is prefixed with an underscore. Doesn't hurt though.